### ISSUE TITLE
Center navbar

### ISSUE THREAD

**Issue by dnestico** [javawizard/ha-navbar-position#6]
Ok so I moved the navbar position but is there a way to center the icons now? Looks much nicer on mobile IMO.

**Comment by Lemonadel** [comment]
Yeah it would be good to have them centered when number is < 7 or 6 whatever it triggering the scroll to appear. 

Thank you @javawizard so much for your work. I still could not understand why it’s not a part of Home Assistant Core as it’s TOP NOTCH and with recent update works flawlessly and truly removed empty space on top after I retriggered this down panel to switch places.
